libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/RouteRefreshControllerTest.kt

```diff
@@ -165,23 +162,23 @@ class RouteRefreshControllerTest {
     }
 
     @Test
-    fun `do not send a request when uuid is empty`() = coroutineRule.runBlockingTest {
+    fun `do not send a request when uuid is empty`() = runBlockingTest {
         val directionsSession = mockk<DirectionsSession>(relaxed = true)
```

Comment:
Doesn't it make sense to mock just RouteRefresh now and rename the variable? Same goes for the other tests.